R/deduplicate.R

```diff
@@ -79,6 +79,13 @@ orderly_deduplicate_info <- function(config) {
                           fsep = "/")
 
   paths <- file.path(config$root, "archive", files$path)
+  exists <- vlapply(paths, file.exists)
```

Comment:
I wonder about looking for the existence of `file.path(config$root, "metadata")` (or perhaps that there is at least one file within that directory); that would be stricter and easier to check?  What do you think?

Reply:
If we want to fail with a specific message for people who have done a pull with `recursive = FALSE` then this works. People could still be in a state where they get a not very informative error later though from the stopifnot call. Say if they have deleted a report but not rebuilt the database.